### PR TITLE
🛠️🌸 `Components`: Install `Lookbook` for Component Preview

### PR DESCRIPTION
**Pull request by zspencer**
- https://github.com/zinc-collective/convene/issues/1187

Now we get even *shinier* components! Such shiny! Very browse! Much wow!

I went with the 2.0 beta because I figured it's close to prod.
Screenshot 2023-03-06 at 4 24 40 PM

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -41,7 +41,9 @@ gem "redcarpet", "~> 3.6"
 # Breadcrumbs!
 gem "gretel", "~> 4.4"
 # Better UI components
+gem "lookbook", ">= 2.0.0.beta.4"
```
Comment (KellyAH) [inline Gemfile:44]: beta? is that stable?

**Review by anaulin — APPROVED** [review]
Hopefully this newer Lookbook version works better with Zeitwerk!

**Comment by zspencer** [comment]
Yea, I'm worried about the autoloading; but 2.0 seems to do a bit better than the 1.7 I've used in other projects./

File: Gemfile (hunk 1 above)
Comment (zspencer) [reply Gemfile:44]: I went with "stable enough" since it's a 2.0 beta.

**Pull request merged**